src/main/java/reposense/authorship/FileInfoAnalyzer.java

```diff
@@ -50,7 +51,7 @@ public static FileResult analyzeFile(RepoConfiguration config, FileInfo fileInfo
             CheckStyleParser.aggregateStyleIssue(fileInfo, config.getRepoRoot());
         }
         if (config.isAnnotationOverwrite()) {
-            AnnotatorAnalyzer.aggregateAnnotationAuthorInfo(fileInfo, config.getAuthorAliasMap());
+            AnnotatorAnalyzer.aggregateAnnotationAuthorInfo(fileInfo, config.getAuthorEmailsAndAliasesMap());
```

Comment:
Should we keep email and aliases separate instead?

Reply:
currently it does not provide any benefits to splitting them up, it will only bloats up the code base hence i combine them map-wise.